src/net.cpp

```diff
@@ -2185,11 +2185,36 @@ void CConnman::WakeMessageHandler()
 
 void CConnman::ThreadDNSAddressSeed()
 {
+    constexpr int TARGET_OUTBOUND_CONNECTIONS = 2;
+    int outbound_connection_count = 0;
+
+    auto start = NodeClock::now();
+    if (gArgs.IsArgSet("-seednode")) {
```

Comment:
Nit:
```suggestion
    if (gArgs.IsArgSet("-seednode")) {
        auto start = NodeClock::now();
```

Reply:
I'm open to being wrong about this if you think `start` may be useful outside of the `-seednode` logic. If we refactored the dns seed `seeds_wait_time` sleep loop to resemble this part, maybe we'd want to reinitialize the variable anyways:
```diff
  const std::chrono::seconds seeds_wait_time = (addrman.Size() >= DNSSEEDS_DELAY_PEER_THRESHOLD ? DNSSEEDS_DELAY_MANY_PEERS : DNSSEEDS_DELAY_FEW_PEERS);
+ auto start = NodeClock::now();
```

Reply:
I think this makes sense. I wouldn't repurpose `seeds_wait_time` though. Despite both being `chrono::seconds`, they are semantically different (one is supposed to be a timestamp, while the other is a delta to be added to a timestamp, so even though we may simplify one variable, the code may be harder to read)